src/explore-education-statistics-frontend/src/modules/subscriptions/components/VerificationErrorMessage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Link from '@frontend/components/Link';
+
+interface VerificationErrorMessageProps {
+  slug: string;
+}
+
+const VerificationErrorMessage = ({ slug }: VerificationErrorMessageProps) => {
+  return (
+    <div className="govuk-panel__body">
+      <p>
+        Your subscription verification token has expired. You can try again by
+        re-subscribing from the{' '}
+        <Link to={`/find-statistics/${slug}`}>publication's main screen.</Link>
```

Comment:
Cam will be happy with this. A link to the applicable publication was a nice-to-have when the verification error page was added before, but the data wasn't available because the token linking it with a publication no longer existed.